portal/src/main/webapp/WEB-INF/jsp/global/header_bar.jsp

```diff
@@ -49,6 +49,9 @@
             "/" + GlobalProperties.getRightLogo() : GlobalProperties.getRightLogo();
     pageContext.setAttribute("rightLogo", rightLogo);
 %>
+<c:url var="samlLogoutUrl" value="/saml/logout">
+    <c:param name="local" value="${GlobalProperties.getSamlIsLogoutLocal()}" />
```

Comment:
I understand this change is not part of this PR, but It looks like global logout is the default in GlobalProperties.  I don't think we want this as the default behavior.  From what I understand, a global logout would terminate other SP sessions, not just the cBioPortal session.  Unless there are objections, can we update the default in this PR?

Reply:
I don't want this comment to get lost, but I will have to change this particular line of code for other reasons (Tomcat 7 compatibility, same issue as in https://github.com/cBioPortal/cbioportal/pull/3107). @pieterlukasse, I think you know more about the considerations Ben mentions here than I do.